src/lib/githubDataManager.ts

```
@@ -0,0 +1,357 @@
+  SETS_DATA: 'pokemon_tcg_sets_data',
+  CARDS_DATA_PREFIX: 'pokemon_tcg_cards_',
+  LAST_UPDATE: 'pokemon_tcg_last_update',
+};
+
+// Check for updates every 24 hours
+const UPDATE_INTERVAL = 24 * 60 * 60 * 1000; // 24 hours in milliseconds
+
+/**
+ * Fetches the sets data from local data or GitHub
+ */
+export async function fetchSetsData(): Promise<any[]> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)